social-network/src/components/ArticleAuthor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from 'react-bootstrap/Image'
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+export default class AuthorArticle extends React.Component {
+    static propTypes = {
+        author: PropTypes.object,
```

Comment:
Not a huge deal, but another way to write this could be using `PropTypes.shape`. e.g.

```js
author: PropTypes.shape({
  name: PropTypes.string,
  image: PropTypes.string
})
```

Since the component is pretty small and modular, it's not completely necessary though.